### DISCUSSION TITLE
How to increase use of more CPU Cores ?

### DISCUSSION THREAD

**Discussion by OmGuptaIND** [uNetworking/uWebSockets.js#1008] [Q&A] [answered]
https://github.com/uNetworking/uWebSockets.js/discussions/748#discussioncomment-2852483

I didnt get this it just uses one core. Cause of Node.js using more core and single threaded

**Comment by uNetworkingAB** [comment, marked as answer]
There are examples, blog posts and user manual describing it. You can start with running this example https://github.com/uNetworking/uWebSockets.js/blob/master/examples/WorkerThreads.js

**Reply by uNetworkingAB** [reply]
Here is a blog post with 4 CPU-cores https://medium.com/@unetworkingab/serving-100k-requests-second-from-a-fanless-raspberry-pi-4-over-ethernet-fdd2c2e05a1e
